src/components/Company/FederalTaxes/Form.tsx

```diff
@@ -56,14 +56,11 @@ export function Form() {
             }}
           />
         }
-        control={control}
         isRequired
-        inputProps={{
-          placeholder: placeholderEin,
-          onChange: event => {
-            setValue('federalEin', normalizeEin(event.target.value))
-          },
+        onChange={event => {
+          setValue('federalEin', normalizeEin(event.target.value))
```

Comment:
We could update this to use `transform` instead

Reply:
Fixed